BeeSwiftToday/TodayViewController.swift

```diff
@@ -22,14 +22,14 @@ class TodayViewController: UIViewController {
     override func viewDidLoad() {
         NotificationCenter.default.addObserver(self, selector: #selector(self.updateDataSource), name: UserDefaults.didChangeNotification, object: nil)
         
+        self.updateDataSource()
+        
         self.preferredContentSize = CGSize.init(width: 0, height: self.rowHeight)
         
         if self.goalDictionaries.count > 1 {
```

Comment:
agreed it makes sense to calculate this after `updateDataSource`, just this commit is not 'defensive' enough